api/src/main/java/org/commonjava/indy/spi/pkg/ContentQuality.java

```diff
@@ -0,0 +1,8 @@
+package org.commonjava.indy.spi.pkg;
+
+/**
+ */
+public enum ContentQuality
+{
+    METADATA, SNAPSHOT, RELEASE
```

Comment:
If we determine that "unspecified" / null == RELEASE, it might not be meaningful to have RELEASE in here.